src/Array.mo

```diff
@@ -21,33 +21,35 @@ import { todo } "Debug";
 
 module {
 
-  /// Create an empty array (equivalent to `[]`).
+  /// Creates an empty array (equivalent to `[]`).
   public func empty<T>() : [T] = [];
 
-  /// Create an array with `size` copies of the initial value.
+  /// Creates an array containing `item` repeated `size` times.
   ///
   /// ```motoko include=import
-  /// let array = Array.init<Nat>(4, 2);
+  /// let array = Array.repeat<Nat>("Echo", 3);
+  /// assert array == [var "Echo", "Echo", "Echo"];
   /// ```
   ///
   /// Runtime: O(size)
   ///
   /// Space: O(size)
-  public func init<T>(size : Nat, initValue : T) : [T] = Prim.Array_tabulate<T>(size, func _ = initValue);
+  public func repeat<T>(item : T, size : Nat) : [T] = Prim.Array_tabulate<T>(size, func _ = item);
```

Comment:
For some reason, I'd expect the size to come first..

Reply:
An alternative name might be `replicate` to rhyme with `tabulate`, but maybe repeat is established in other langs.

Reply:
`repeat()` comes from Java and JS strings, where I've extended this naming convention to other modules for consistency.

Reply:
As in let's stick with this.

Reply:
Though I guess there is some risk that people will rename from init without swapping argument order...